pages/login.vue

```diff
@@ -86,11 +91,15 @@ export default {
   justify-content: center;
 }
 .v-form {
-  padding: 10px !important;
+  padding: 18px !important;
```

Comment:
please don't use importants! (to be fair it was already there, but try if you can to remove it)

Reply:
would also recommend using `rem` or `em` instead of px. 
This helps adjust text size for the visually impaired.

Reply:
- removed `!important`
- replaced `px` with `rem`